### DISCUSSION TITLE
MacOS Speed

### DISCUSSION THREAD

**Discussion by Mellvik** [86Box/86Box#2525] [Q&A] [answered]
[newbie question] 
Curiosity: Is 86box being intentionally throttled on macos? 

Running various 286 and 386 system emulations I'm getting speeds an order of magnitude slower than the real thing. And the real thing is on my desk.

-M

**Comment by dhrdlicka** [comment, marked as answer]
No, there shouldn't be any throttling of any kind. Make sure you pick the same clock rate in settings and that the emulation runs at 100% speed.
